docs/first-order/modality_alignment.ipynb

```diff
@@ -0,0 +1,600 @@
+{
```

Comment:
<p>Should we upgrade the Dandi-cli? Since Dandi is an API, staying too far behind can be a problem</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/223/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>